### DISCUSSION TITLE
Anyone know how WSASettings.exe is intended to be launched on the commandline?

### DISCUSSION THREAD

**Discussion by anonhostpi** [LSPosed/MagiskOnWSALocal#704] [Q&A]
I noticed that running WSASettings.exe straight from the build files, it gives me a .dll missing error.

I opened WSASettings using the start menu and it opens fine. When I did that I ran (Get-Process wsasettings).CommandLine and found that it was run like this:

```powershell
C:\...\WsaSettings.exe -ServerName:SettingsApp.AppXe5xfar8kfrnsjyxycsqtgfxc6xgc0zed.mca
```

Any idea where that ServerName comes from?

**Comment by mohrpw** [comment]
For anyone else encountering this problem, I found a workaround.

Run CTRL + R -> `wsa-settings:`
